td-shim/src/bin/td-shim/td/tdx.rs

```diff
@@ -11,39 +11,8 @@ extern "win64" {
 
 const EXTENDED_FUNCTION_INFO: u32 = 0x80000000;
 const EXTENDED_PROCESSOR_INFO: u32 = 0x80000001;
-const HARDWARE_ENABLED_EXECUTION_DISABLE: bool = false;
```

Comment:
The issue could also get fixed by changing "HARDWARE_ENABLED_EXECUTION_DISABLE" to true, as in #90 :)

It would be great to document that `excution_disable` is actually enabled by hardware.

Reply:
Having excution_disable() API is misleading. I don't think we need this function.
Yes, we should document it.